lib/ExpandingTextArea/index.js

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class ExpandingTextArea extends Component {
+  static propTypes = {
+    placeholder: PropTypes.string.isRequired,
+    handleOnChange: PropTypes.func,
+    value: PropTypes.string,
+    focusOnMount: PropTypes.bool,
+    setValue: PropTypes.bool
+  };
+
+  static defaultProps = {
+    handleOnChange: () => {},
+    value: '',
+    focusOnMount: false,
+    setValue: false
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = { inputValue: '', rows: 1, rowSize: 0, padding: 0 };
```

Comment:
Is `rowSize` used as the physical size of the rows or is it the count of rows e.g. `rowCount`?

Reply:
Yeah thats confusing, I'll rename so they're more descriptive